temp/patientSignUp.html

```diff
@@ -48,13 +48,14 @@
                         <ul class="nav">
                             <li class="scroll-to-section"><a href="index.html" class="active">Home</a></li>
                             <li class="scroll-to-section"><a href="index.html#about">About</a></li>
+                            <li class="scroll-to-section"><a href="/templates/demo.html">Demo</a></li>
```

Comment:
We should definitely look into extracting the header into its own component and import it as a single element in these pages so that when we add/modify navigation items, we don't need to individually add to each page.

Reply:
Opened this issue for this:
https://github.com/mellychat/webapp/issues/75